strawberry/experimental/pydantic/error_type.py

```diff
@@ -16,25 +15,26 @@
 )
 
 from pydantic import BaseModel
-from pydantic.utils import lenient_issubclass
 
 from strawberry.auto import StrawberryAuto
 from strawberry.experimental.pydantic.utils import (
     get_private_fields,
     get_strawberry_type_from_model,
     normalize_type,
 )
+from strawberry.experimental.pydantic.v2_compat import (
```

Comment:
```suggestion
from strawberry.experimental.pydantic._compat import (
```

can we call it `_compat`? 😊